resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/ui/DriverPage.scala

```diff
@@ -153,6 +153,9 @@ private[ui] class DriverPage(parent: MesosClusterUI) extends WebUIPage("driver")
     <tr>
       <td>Memory</td><td>{driver.mem}</td>
     </tr>
+    <tr>
+      <td>Queue</td><td>{driver.conf.get("spark.mesos.dispatcher.queue", config.DEFAULT_QUEUE)}</td>
```

Comment:
Instead of this pattern, could you try `DISPATCHER_QUEUE.defaultValue.get`?

Reply:
I opted for `DISPATCHER_QUEUE.defaultValueString`.